README.md

```diff
@@ -479,52 +479,56 @@ performance.
 
 ##### Decoupled Responses
 
-It is also possible for a backend to send multiple responses for a
-request or not send any responses for a request. A backend may also
+It is also possible for a backend to send multiple responses
+for a request. A backend may also
 send responses out-of-order relative to the order that the request
 batches are executed. Such backends are called *decoupled* backends.
-The decoupled backends use one `ResponseFactory` object per request to keep
-creating and sending any number of responses for the request. For this
+
+The decoupled backends use one `ResponseFactory` object per request to
```

Comment:
Add a section before this clarifying what we mean by the response in this context. 
Also point users to this section: https://github.com/triton-inference-server/server/blob/main/docs/user_guide/decoupled_models.md#knowing-when-a-decoupled-inference-request-is-complete
On how these empty final responses can be detected on the client side. Basically, we should clarify the meaning of response from backend's context and the client context and how they are related.

Reply:
@tanmayv25 - I think I would avoid too much on this point here as it can confuse the issue - meaning how a client detects it is seperate from what the requirement on the backend is. We can add a pointer for further information - but I wouldn't try to describe how they relate here if it becomes confusing. 

I think the main thing to clarify here is just the exact requirement for the backend.

Does that make sense?

Reply:
I understand. Marking this issue as resolved.